modules/core/src/main/java/org/apache/ignite/internal/processors/tracing/SpanManager.java

```diff
@@ -48,7 +48,7 @@ default Span create(@NotNull SpanType spanType) {
      * @param serializedParentSpan Parent span as serialized bytes.
      * @return Created span.
      */
-    Span create(@NotNull SpanType spanType, @Nullable byte[] serializedParentSpan);
+    Span create(@NotNull SpanType spanType, byte @Nullable [] serializedParentSpan);
```

Comment:
From the language point of view, it has the same meaning, right? So what is the reason for this change?

Reply:
Just a suggestion of IDEA. I suppose it uses generic pattern for all arrays. In this case you're right it doesn't matter. But in case of, for example, Person[], moving `@Nullable` makes sence. Nullable array and array of nullables are indeed different.

In this particular case I agree, I overcomplicated it. Will fix it.